v0/destinations/webengage/data/WEBENGAGEIdentifyConfig.json

```diff
@@ -0,0 +1,81 @@
+[
+  {
+    "destKey": "userId",
+    "sourceKeys": "userIdOnly",
+    "sourceFromGenericMap": true,
+    "required": false
+  },
+  {
+    "destKey": "anonymousId",
+    "sourceKeys": "anonymousId",
+    "required": false
+  },
+  {
+    "destKey": "firstName",
+    "sourceKeys": "firstName",
+    "sourceFromGenericMap": true,
+    "required": false
+  },
+  {
+    "destKey": "lastName",
+    "sourceKeys": "lastName",
+    "sourceFromGenericMap": true,
+    "required": false
+  },
+  {
+    "destKey": "birthDate",
+    "sourceKeys": ["traits.birthDate", "context.traits.birthDate"],
+    "required": false
+  },
+  {
+    "destKey": "gender",
+    "sourceKeys": ["traits.gender", "context.traits.gender"],
+    "required": false
+  },
+  {
+    "destKey": "email",
+    "sourceKeys": "email",
+    "sourceFromGenericMap": true,
+    "required": false
+  },
+  {
+    "destKey": "phone",
+    "sourceKeys": "phone",
+    "sourceFromGenericMap": true,
+    "required": false
+  },
+  {
+    "destKey": "emailOptIn",
+    "sourceKeys": ["traits.emailOptIn", "context.traits.emailOptIn"],
+    "required": false
+  },
+  {
+    "destKey": "smsOptIn",
+    "sourceKeys": ["traits.smsOptIn", "context.traits.smsOptIn"],
+    "required": false
+  },
+  {
+    "destKey": "whatsappOptIn",
+    "sourceKeys": ["traits.whatsappOptIn", "context.traits.whatsappOptIn"],
+    "required": false
+  },
+  {
+    "destKey": "company",
+    "sourceKeys": ["traits.company", "context.traits.company"],
+    "required": false
+  },
+  {
+    "destKey": "hashedEmail",
+    "sourceKeys": ["traits.hashedEmail", "context.traits.hashedEmail"],
```

Comment:
Once the user set hashed email and hashed phone, then it cannot be changed by sending the email and phone number, it could only be changed by hashed email and hashed phone. So we takes both and mapped them with their respective fields.